src/utils.c

```diff
@@ -280,3 +280,25 @@ SEXP islockedR(SEXP DT) {
   return ScalarLogical(islocked(DT));
 }
 
+
+// To see if the character vector contains non-ASCII strings not encoded in UTF-8
+bool need2utf8(SEXP x) {
+  const int xlen = length(x);
+  for (int i=0; i<xlen; i++)
+    if (NEED2UTF8(STRING_ELT(x, i))) 
+      return(true);
+    return(false);
```

Comment:
looks like `return(false)` should be two whitespace less, no? `for` could get `{` for readability

Reply:
you are right. thanks.